cms/djangoapps/xblock_config/models.py

```diff
@@ -26,3 +27,55 @@ def asides_enabled(cls, block_type):
         """
         studio_config = cls.current()
         return studio_config.enabled and block_type not in studio_config.disabled_blocks.split()
+
+
+class CourseEditLTIFieldsEnabledFlag(ConfigurationModel):
+    """
+    Enables the editing of "request username" and "request email" fields
+    of LTI consumer for a specific course.
+    """
+    KEY_FIELDS = ('course_id',)
+
+    # The course that this feature is attached to.
+    course_id = CourseKeyField(max_length=255, db_index=True)
+
+    @classmethod
+    def lti_access_to_learners_editable(cls, course_id, is_already_sharing_learner_info):
+        """
+        Looks at the currently active configuration model to determine whether
+        the feature that enables editing of "request username" and "request email"
+        fields of LTI consumer is available or not.
+
+        Backwards Compatibility:
+        Enable this feature for a course run who was sharing learner username/email
+        in the past.
+
+        Arguments:
+            course_id (CourseKey): course id for which we need to check this configuration
+            is_already_sharing_learner_info (bool): indicates whether LTI consumer is
+            already sharing edX learner username/email.
+        """
+        course_specific_config = (CourseEditLTIFieldsEnabledFlag.objects
+                                  .filter(course_id=course_id)
+                                  .order_by('-change_date')
+                                  .first())
+
+        if is_already_sharing_learner_info:
+            # Enable the feature for this course run if it is already sharing the learner information and
+            # course-specific configuration is not present for it - in that case, it is going to be an
+            # existing course run.
+            if not course_specific_config:
+                CourseEditLTIFieldsEnabledFlag.objects.create(course_id=course_id, enabled=True)
+                return True
+
+        return course_specific_config.enabled if course_specific_config is not None else False
```

Comment:
The suggested condition should work. But if you want to keep the existing condition then condition should be 
`return course_specific_config.enabled if course_specific_config else False` because as per my understanding `course_specific_config` will be never set to `None`.